jcl/src/java.management/share/classes/java/lang/management/PlatformLoggingMXBean.java

```diff
@@ -44,10 +45,17 @@
  * 
  * @since 1.5
  */
+/*[IF JAVA_SPEC_VERSION >= 19]*/
+@SuppressWarnings("doclint:reference")
```

Comment:
Is this absolutely necessary?

Reply:
Yes, it is required, otherwise `warning: module for reference not found: java.logging` can cause build failure due to `-Werror` specified. This is RI behaviours as well.

Reply:
Apparently, the RI has expressed their javadoc so as not to trigger warnings. We should attempt to do the same rather than just suppress the warning.

Reply:
My experiment shows otherwise, what RI javadoc expression avoided the warnings?

Reply:
Ok, I see the `SuppressWarnings` upstream. Perhaps it's unavoidable because the target module is not a prerequisite of `java.management`.